extmod/network_ninaw10.c

```diff
@@ -197,14 +197,14 @@ static mp_obj_t network_ninaw10_active(size_t n_args, const mp_obj_t *args) {
             int error = 0;
             if ((error = nina_init()) != 0) {
                 mp_raise_msg_varg(&mp_type_OSError,
-                    MP_ERROR_TEXT("Failed to initialize Nina-W10 module, error: %d\n"), error);
+                    MP_ERROR_TEXT("failed to initialize Nina-W10 module, error: %d\n"), error);
```

Comment:
please remove the newline at the end (when errors are printed, newlines are added)